bundles/core/org.eclipse.smarthome.core.test/src/test/java/org/eclipse/smarthome/core/internal/items/ItemRegistryOSGiJavaTest.java

```diff
@@ -194,9 +195,13 @@ public void testRemoveNonExistingTag() throws Exception {
         assertTagsInItem(ITEM_NAME, "foo");
     }
 
-    @Test(expected = IllegalArgumentException.class)
+    @Test
     public void testRemoveTagFromNonExistingItem() throws Exception {
+        prepareMetadata(ITEM_NAME, "hello");
+
         itemRegistry.removeTag(ITEM_NAME, "hello");
+
+        assertTagsInMetadata(ITEM_NAME);
```

Comment:
Is this assertion correct? I mean, if you have removed the tag, the assertion will probably fail.

Reply:
I assume (WRT the method name) you would like to remove the item and not the tag, correct?

Reply:
No, he doesn't pass any `tags` to the `assertTagsInMetadata` method and thus this method verifies that for the given item there are no tags. So the assertion is correct.